skrf/io/mdif.py

```diff
@@ -365,7 +365,9 @@ def write(ns : NetworkSet,
               filename : str,
               values: dict | None = None,
               data_types: dict | None = None,
-              comments = None):
+              ads_compatible: bool = True,
+              comments: str | None = None,
+              skrf_comment: bool = True):
```

Comment:
please keep the order and do not insert `ads_compatible` before `comments`.
Also use `*` to seperate boolean comments from the rest and make them keyword only for clarification.

https://peps.python.org/pep-3102/

Same below.